src/Features/Core/IntroduceVariable/AbstractIntroduceVariableService.cs

```diff
@@ -134,10 +117,39 @@ protected virtual bool BlockOverlapsHiddenPosition(SyntaxNode block, Cancellatio
                     }
                 }
             }
+            else if (state.InExpressionBodiedMemberContext)
+            {
+                await CreateConstantFieldActionsAsync(state, actions, cancellationToken).ConfigureAwait(false);
+                actions.Add(CreateAction(state, allOccurrences: false, isConstant: state.IsConstant, isLocal: true, isQueryLocal: false));
+                actions.Add(CreateAction(state, allOccurrences: true, isConstant: state.IsConstant, isLocal: true, isQueryLocal: false));
+            }
```

Comment:
@dpoeschl Actions are created differently based on the context we're in.. What would the else (default) case do, if it can't construct or offer a generic action? Is throwing worth here?